File: Web/RMA-Web-Shell/projects/medicare/src/app/medical-invoice-manager/models/modifier-input.ts
import { InvoiceLineDetails } from "projects/medicare/src/app/medical-invoice-manager/models/medical-invoice-line-details";

export class ModifierInput{
    ModifierCode: string;
    ModifierDescription: string;
    ModifierServiceDate: Date;
    HealthCareProviderId: number;
    PractitionerTypeId: number;
    TariffCode: string;
    TariffServiceDate: Date;
    TariffQuantity: number;
    TariffAmount: number;
    TariffDiscount: number;
    TariffBaseUnitCostTypeId: number;
    TariffTypeId: number;
    PublicationId: number;
    IsModifier: boolean;
    UnitPrice: number;
    PreviousLinesTotalAmount: number;
    PreviousInvoiceLine: InvoiceLineDetails;
    PreviousInvoiceLines: InvoiceLineDetails[];
    TimeUnits: number;
    SectionNo: string;
    RecommendedUnits: number;
    ReductionCodes: string[]
}

File: Service/MediCare/RMA.Service.MediCare.Contracts/Entities/Medical/ModifierInput.cs
using System;
using System.Collections.Generic;

namespace RMA.Service.MediCare.Contracts.Entities.Medical
{
    public class ModifierInput
    {
        public string ModifierCode { get; set; }
        public string ModifierDescription { get; set; }
        public DateTime ModifierServiceDate { get; set; }
        public int HealthCareProviderId { get; set; }
        public int PractitionerTypeId { get; set; }
        public string TariffCode { get; set; }
        public DateTime TariffServiceDate { get; set; }
        public decimal TariffQuantity { get; set; }
        public decimal TariffAmount { get; set; }
        public decimal TariffDiscount { get; set; }
        public int TariffBaseUnitCostTypeId { get; set; }
        public int TariffTypeId { get; set; }
        public int PublicationId { get; set; }
        public bool IsModifier { get; set; }
        public decimal UnitPrice { get; set; }
        public decimal PreviousLinesTotalAmount { get; set; }
        public InvoiceLineDetails PreviousInvoiceLine { get; set; }
        public List<InvoiceLineDetails> PreviousInvoiceLines { get; set; }
        public int TimeUnits { get; set; }
        public string SectionNo { get; set; }
        public decimal RecommendedUnits {  get; set; }
        public List<string> ReductionCodes { get; set; }
    }
}
